hw/ip/lc_ctrl/data/lc_ctrl_state.hjson

```diff
@@ -0,0 +1,40 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+//
+// Use the gen-lc-state-enc.py script to regenerate the SV package:
+//
+// $ cd ${PROJ_ROOT}/hw/ip/lc_ctrl/util/
+// $ ./gen-lc-state-enc.py
+//
+
+{
+    // Seed to be used for generation of partition item default values.
+    // Can be overridden on the command line with the --seed switch.
+    seed: 10167336684108184581
+
+    // Secded used in prim_generic_otp.sv
+    secded : {
```

Comment:
can you think of any way that would quickly trigger an error if otp side changed somehow and we forgot to modify this one?

Reply:
i guess maybe sims would just fail eventually...since the lc state would just be garbage?

Reply:
Yeah good point. Once we can preload the OTP in simulations, this should become apparent in top-level simulations during the power-up initialization, since the ECC will be wrong for most life cycle words.
At the block-level however, I think this will not be visible at the moment, since the LC controller is tested separately from the OTP memory (and hence the ECC logic).

Concerning the different ECC versions (open vs closed), I have not yet added a mechanism to support multiple OTP macros / ECC polynomials yet. There are a couple of options we can consider, but I am not entirely sure which one would be best. E.g.:
1. use the `ImplXY` parameter inside the package to select constants from different sources. this must be done in a way such that the ASIC version is always located in the `else` leg of ternaries, generator blocks or preprocessor ifdef blocks, since we do not want to expose the technology string.
1. use the fusesoc flag mechanism to select either the open-source or closed-source package, where both packages have the same name.
1. auto-generate the correct package using a generator script, whenever the top-level is built.

Also, it is not entirely clear to me how we are going to include proprietary simulation models, hence I thought I'd wait until we sorted that one out.